examples/hf/pippy_megatronBert.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 4 pippy_bert.py
```

Comment:
this should be "pippy_megatronbert.py" instead of pippy_bert.py.

Reply:
Fixed, thanks!